CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 #### Fixes
 
 * Your contribution here.
+* [#2089](https://github.com/ruby-grape/grape/pull/2089): Update README to fix 404 error issue mounting grape using rack::cascade - [@jonmchan](https://github.com/jonmchan).
```

Comment:
Rack::Cascade is capitals and quote it as `Rack::Cascade`

Error and issue is redundant ;) Maybe "Specify order of mounting Grape with `Rack::Cascade` in README"?